db/models/storePostgresDenormal.js

```diff
@@ -0,0 +1,12 @@
+const { Client } = require('pg');
+
+const client = new Client();
+client.connect();
+
+const findOne = id => (
```

Comment:
Between this file, and the storePostgres.js file, your findOne functions are formatted differently, I would consider making them consistent.

Reply:
Currently they are formatted differently because I need to query two different tables for the normal version (denormal queries one table). Do you mean I should combine my two queries to one?